docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
+--------------------
+
+Configuration properties in Trino take values of different types, some standard
+and some non-standard. Supported values may be further limited on a per-property
+basis.
+
+.. _prop-type-boolean:
+
+``boolean``
+^^^^^^^^^^^
+
+Properties of a ``boolean`` type support two values, ``true`` or ``false``
+in lowercase.
+
+.. _prop-type-data-size:
+
+``data size``
+^^^^^^^^^^^^^
+
+Properties of a ``data size`` type support values describing an amount of
+data, measured in byte-based units. For example, the value ``6GB`` describes
+six gigabytes.
+
+The ``data size`` type supports the following units:
+
+* ``B``: Bytes
+* ``kB``: Kilobytes
+* ``MB``: Megabytes
+* ``GB``: Gigabytes
+* ``TB``: Terabytes
+* ``PB``: Petabytes
+
+Some ``data size`` properties also support ``unlimited`` as a value.
```

Comment:
Which ones are these - I'd like to take a look to introduce some consistency.

Reply:
Is it truly Megabytes  .. like 1 Megabyte is 1, 024 Kilobytes .. or is it MByte .. like 1000 KByte or 1000000 Byte
-- | --

Reply:
@mosabua good point

here:
1KB is 1024 bytes
1MB is 1024 KB

@hashhar "unlimited" should NOT be considered part of data type.

Reply:
The ``http-server.log.max-size`` property (in properties-logging.rst) which is a data size type has a default value of "unlimited". That one exception is reason enough to document.

Reply:
Removed the note about ``unlimited`` from the data-size definition, instead making a note of it in the one property that seems to use it.